x-pack/plugins/fleet/server/integration_tests/cloud_preconfiguration.test.ts

```diff
@@ -160,7 +160,6 @@ describe('Fleet preconfiguration reset', () => {
             perPage: 10000,
           });
 
-        expect(packagePolicies.total).toBe(3);
```

Comment:
Can you explain to me what these changes in fleet integration tests are? Just curious.